.changeset/wild-cats-hug.md

```diff
@@ -0,0 +1,6 @@
+---
+"@osdk/client.api": minor
+"@osdk/client": minor
+---
+
+Add Exact Distinct Aggregations
```

Comment:
Nit, these are going to be written as bullets on the release notes so are generally better as phrases/sentences than "titles". e.g. `Adds exactDistinct to aggregations`